src/psyclone/transformations.py

```diff
@@ -46,13 +46,14 @@
 from psyclone.psyGen import Transformation, Kern
 from psyclone.errors import InternalError
 from psyclone.psyir.nodes import Schedule
-from psyclone.psyir.symbols import DataSymbol, DataType
+from psyclone.psyir.symbols import DataSymbol
```

Comment:
There is 2 `from psyclone.psyir.symbols` maybe you can merge them?

Reply:
Oh yes, well spotted.